app/src/main/java/com/tolkiana/tacoapp/ProductAdapter.kt

```diff
@@ -16,7 +16,7 @@ class ProductAdapter(private val products: List<Product>): Adapter<ProductAdapte
 
     override fun onBindViewHolder(productHolder: ProductHolder?, position: Int) {
         val product = products[position]
-        productHolder?.productNameTextView?.text = product.getName()
+        productHolder?.productNameTextView?.text = product.name
```

Comment:
I like that getters in Kotlin are more like properties (`getName()` becomes `name`). 🙌

Reply:
:)